pkgs/ffigen/lib/src/code_generator/objc_block.dart

```diff
@@ -308,44 +311,46 @@ abstract final class $name {
     s.write('}\n\n');
 
     // Call operator extension method.
-    s.write('''
```

Comment:
Unfortunately, I need to be able to call blocks e.g.

```dart
      ncb.NSURLSessionDataDelegate
          .URLSession_dataTask_didReceiveResponse_completionHandler_
          .implementAsListener(protoBuilder,
              (nsSession, nsDataTask, nsResponse, nsCompletionHandler) {
        final exactResponse = URLResponse._exactURLResponseType(nsResponse);
        final disposition = onResponse(
            URLSession._(nsSession,
                isBackground: isBackground, hasDelegate: true),
            URLSessionTask._(nsDataTask),
            exactResponse);
        nsCompletionHandler.call(disposition);
      });
```

Reply:
@brianquinlan Those errors look like you're not using the version of package:objective_c from this branch.

I'll work on supporting the call operator today.

Reply:
Added support for the call operator